src/expandableCalendar/index.tsx

```diff
@@ -269,8 +269,8 @@ const ExpandableCalendar = forwardRef<ExpandableCalendarRef, ExpandableCalendarP
   }, [isOpen, headerHeight]);
 
   const containerStyle = useMemo(() => {
-    return [allowShadow && style.current.containerShadow, propsStyle, headerHeight === 0  && style.current.hidden, {overflow: 'hidden'} as const];
-  }, [allowShadow, propsStyle, headerHeight]);
+    return [allowShadow && style.current.containerShadow, propsStyle, headerHeight === 0 && !screenReaderEnabled  && style.current.hidden, {overflow: 'hidden'} as const];
```

Comment:
I prefer connecting the accessibility calendar to the onLayout then adding this patch